gapic/packaging/api_defaults.yaml

```diff
@@ -16,14 +16,18 @@
 # Provides default values used as metadata when constructing various packages.
 
 author: Google Inc
-description: a grpc-based api
 email: googleapis-packages@google.com
-github_user_uri: https://github.com/googleapis
 homepage: https://github.com/googleapis/googleapis
 license: Apache-2.0
-semver:
+package_version:
```

Comment:
Hm, to me that's more confusing, because `api_version` refers to something unrelated to this configuration value.